ts/parts/Tooltip.ts

```diff
@@ -1689,6 +1729,52 @@ class Tooltip {
         }
     }
 
+    /**
+     * If the `stickOnContact` option is active, this will add a tracker shape.
+     *
+     * @private
+     * @function Highcharts.Tooltip#stickOnContact
+     */
+    private stickOnContact(): void {
```

Comment:
This function name isn't really descriptive anymore. I think a better name would be simply `drawTracker`, as we now have a tracker that is equivalent to the series tracker.